tests/ops/api/v1/endpoints/test_connection_template_endpoints.py

```diff
@@ -1005,7 +1005,7 @@ def test_get_connection_secret_schema_mysql(
                     "type": "string",
                 },
                 "ssh_required": {
-                    "title": "SSH Required",
+                    "title": "SSH required",
```

Comment:
Changing `SSH required` to use sentence-case to be consistent with the other field labels.

Reply:
how dare you!